src/bentoml_cli/models.py

```diff
@@ -233,13 +233,20 @@ def export(model_tag: str, out_path: str) -> None:  # type: ignore (not accessed
 
     @model_cli.command(name="import")
     @click.argument("model_path", type=click.STRING)
-    def import_from(model_path: str) -> None:  # type: ignore (not accessed)
+    @click.option(
+        "--name",
+        type=click.STRING,
+        default=None,
+        help="The name to give to the model in the BentoML store",
+    )
+    def import_from(model_path: str, name: str | None) -> None:  # type: ignore (not accessed)
         """Import a previously exported Model archive file
 
         bentoml models import ./my_model.bentomodel
         bentoml models import s3://mybucket/models/my_model.bentomodel
+        bentoml models import hf:t5-base --name myModelName
```

Comment:
I don't think we should have a custom name support here.

What stops people from doing the following

```bash

bentoml models import s3://mybucket/models/my_model.bentomodel --name ADiffName
```

Importing HF models should just follow a specific format
```prolog
repo--model-name
```

We can save the model format (whether it is PyTorch, Flax, TF) in either labels or context.

Reply:
> We can save the model format (whether it is PyTorch, Flax, TF) in either labels or context.

Can I have a bit more detail on what you mean by this?

Reply:
`pt-meta-llama--Llama-2-13b-chat-hf`
or `flax-google--vit-transformers-384...`